lib/linux.js

```diff
@@ -99,11 +103,12 @@ module.exports = async () => {
 			boundsStdout,
 			stdout
 		});
-		const [memoryUsage, path] = await Promise.all([
-			getMemoryUsageByPid(data.owner.processId),
-			getPathByPid(data.owner.processId)
-		]);
-		data.memoryUsage = memoryUsage;
+		data.memoryUsage = await getMemoryUsageByPid(data.owner.processId);
+		let path = null;
+		try {
+			path = await getPathByPid(data.owner.processId);
+		} catch {}
```

Comment:
Please preserve the concurrent fetching.

Reply:
Absolutely, my mistake. Restored concurrent fetching via `Promise.all` with 92b6d381.